src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker_Patterns.cs

```diff
@@ -149,6 +156,9 @@ public override BoundNode VisitITuplePattern(BoundITuplePattern node)
                         LearnFromNullTest(inputSlot, inputType, ref this.State, markDependentSlotsNotNull: false);
                     }
                     break;
+                case BoundListPattern lp:
+                    // PROTOTYPE(list-patterns)
```

Comment:
I'll refresh PR [https://github.com/dotnet/roslyn/pull/53822](https://github.com/dotnet/roslyn/pull/53822) for nullability of list patterns after this syntax change is merged.